neo/Network/P2P/Payloads/Witness.cs

```diff
@@ -24,16 +25,29 @@ public virtual UInt160 ScriptHash
             }
         }
 
-        public int Size => InvocationScript.GetVarSize() + VerificationScript.GetVarSize();
+        public bool HasScopedHash => Scope == WitnessScope.CustomScriptHash || Scope == WitnessScope.ExecutingGroupPubKey;
```

Comment:
This is very well done Shargon, congratulations. We just need to filter this pubkey here, as it has another size other than Hash160.... I'm not against groupHash concept Hash160(pubkey) or Hash160(21+pubkey+ac concept...), and group becomes a witness :joy: or just keep pubkey, and count bytes correctly ;)